templates/test/module_table.html.ep

```diff
@@ -49,7 +49,11 @@
                             % my $href  = "#step/$module->{name}/$step->{num}";
                             % if ($is_parser_text_result) {
                                 <span title="<%= $title %>" data-href="<%= $href %>" onclick="toggleTextPreview(this)" class="text-result">
-                                    <span class="resborder <%= $resborder %>"><%= $step->{text_data} %></span>
+                                    <span class="resborder <%= $resborder %>"><%= $step->{text_data} %><span class="step_actions" style="float: right; white-space: normal;">
+                                        % stash('moduleid', $module->{name});
```

Comment:
Stashing things within the template code looks wrong. It would make more sense to pass these variables as arguments to the `bug_report_actions` helper and then pass them to `render_to_string`.